docker/docker-compose.yml

```diff
@@ -21,12 +20,12 @@ services:
         - ./${DEVICE}/detectron2/mask_rcnn_R_50_FPN_3x.yaml:/config.yaml
         - ./${DEVICE}/detectron2/start_backend.sh:/start_backend.sh
         - ./${DEVICE}/entrypoint.sh:/entrypoint.sh
+#        - ../rasberry_perception:/catkin_ws/src/rasberry_perception
```

Comment:
I think it needs `../../` instead of `../` as the parent folder is the in the rasberry_perception